### DISCUSSION TITLE
band contrast extrapolations

### DISCUSSION THREAD

**Discussion by amirtopaz** [mtex-toolbox/mtex#1181] [Ask Anything]
Hi everyone,
Is there a way to play with the band contrast command to make good looking maps of band contrasts?

![grain 69 bc](https://user-images.githubusercontent.com/62837798/130574736-959f6a61-8482-455b-aa9c-42a978706048.jpg)

For example, 
I use the code: plot(ebsd,ebsd.bc)
and this is the type of maps that I get.

Is there a way to make it look a little bit more like what I see in the Atzec software when scanning the EBSD map, like this for example?
![bc](https://user-images.githubusercontent.com/62837798/130575338-75b49dde-dd24-4ee2-877d-e5aee9e3764d.JPG)

Thanks,
Amir

**Comment by lfgmorales** [comment]
Are these two the same map?

In any case, if you want to plot bc in grey scale, just chage the colorcode to white to black



mtexColorMap white2black

Cheers

Luiz


Luiz F. G. Morales
ScopeM - ETH Zürich
Sent from iPhone

On 24 Aug 2021, at 09:31, amirtopaz ***@***.***> wrote:

﻿

Hi everyone,
Is there a way to play with the band contrast command to make good looking maps of band contrasts?

[grain 69 bc]<https://user-images.githubusercontent.com/62837798/130574736-959f6a61-8482-455b-aa9c-42a978706048.jpg>

For example,
I use the code: plot(ebsd,ebsd.bc)
and this is the type of maps that I get.

Is there a way to make it look a little bit more like what I see in the Atzec software when scanning the EBSD map, like this for example?
[bc]<https://user-images.githubusercontent.com/62837798/130575338-75b49dde-dd24-4ee2-877d-e5aee9e3764d.JPG>

Thanks,
Amir

—
You are receiving this because you are subscribed to this thread.
Reply to this email directly, view it on GitHub<https://github.com/mtex-toolbox/mtex/discussions/1181>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/ALE5INZK5RTV5TEIQPYCN6DT6NDENANCNFSM5CWG7FSQ>.

[ { ***@***.***": "http://schema.org", ***@***.***": "EmailMessage", "potentialAction": { ***@***.***": "ViewAction", "target": "https://github.com/mtex-toolbox/mtex/discussions/1181", "url": "https://github.com/mtex-toolbox/mtex/discussions/1181", "name": "View Discussion" }, "description": "View this Discussion on GitHub", "publisher": { ***@***.***": "Organization", "name": "GitHub", "url": "https://github.com" } } ]

**Comment by amirtopaz** [comment]
Hi Luis, 
Thanks for the quick reply.
This isn't the same map. I just put these maps together for the example

Is there a way to retrieve the band contrast map like what I get in Aztec? or is it only this pixeled map that is available?

**Comment by lfgmorales** [comment]
Hi Amir

The BC map „quality“ is directly related to your stepsize, so it seems the stepsize in your colored map, the stepsize is too large, or you have selected just a tiny area of a larger map, and because og that it looks pixelated. Is that possible?

Cheers

Luiz

Luiz F. G. Morales
ScopeM - ETH Zürich
Sent from iPhone

On 24 Aug 2021, at 12:22, amirtopaz ***@***.***> wrote:

﻿

Hi Luis,
Thanks for the quick reply.
This isn't the same map. I just put these maps together for the example

Is there a way to retrieve the band contrast map like what I get in Aztec? or is it only this pixeled map that is available?

—
You are receiving this because you commented.
Reply to this email directly, view it on GitHub<https://github.com/mtex-toolbox/mtex/discussions/1181#discussioncomment-1226740>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/ALE5IN6WB2YUZIBBKSRJQULT6NXGFANCNFSM5CWG7FSQ>.

[ { ***@***.***": "http://schema.org", ***@***.***": "EmailMessage", "potentialAction": { ***@***.***": "ViewAction", "target": "https://github.com/mtex-toolbox/mtex/discussions/1181#discussioncomment-1226740", "url": "https://github.com/mtex-toolbox/mtex/discussions/1181#discussioncomment-1226740", "name": "View Discussion" }, "description": "View this Discussion on GitHub", "publisher": { ***@***.***": "Organization", "name": "GitHub", "url": "https://github.com" } } ]
